helm/values.schema.json

```diff
@@ -0,0 +1,1713 @@
+                    ],
+                    "items": {
+                      "$id": "#/properties/validators/items/anyOf/0/properties/trust_roots",
+                      "anyOf": [
+                        {
+                          "$id": "#/properties/validators/items/anyOf/0/properties/trust_roots/anyOf/0",
+                          "type": "object",
+                          "title": "The first anyOf schema",
+                          "properties": {
+                            "name": {
+                              "$id": "#/properties/validators/items/anyOf/1/properties/trust_roots/anyOf/0/properties/name",
+                              "type": "string",
+                              "title": "The trust root's name schema",
+                              "description": "Name of the trust root",
+                              "examples": [
+                                "securesystemsengineering-official"
+                              ]
+                            },
+                            "key": {
+                              "$id": "#/properties/validators/items/anyOf/1/properties/trust_roots/anyOf/0/properties/key",
+                              "type": "string",
+                              "title": "The trust root's key schema.",
+                              "pattern": "(?:-+BEGIN\\sPUBLIC\\sKEY[-]+)\n(?:(?:[A-Za-z0-9+\\/\\s])*={0,4})\n(?:-+END\\sPUBLIC\\sKEY[-]+)",
+                              "description": "Public ECDSA key used for verification of signatures on container images whose policy matches this trust root.",
+                              "examples": [
+                                "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEsx28WV7BsQfnHF1kZmpdCTTLJaWe\nd0CA+JOi8H4REuBaWSZ5zPDe468WuOJ6f71E7WFg3CVEVYHuoZt2UYbN/Q==\n-----END PUBLIC KEY-----\n"
+                              ]
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "key"
+                          ],
+                          "minItems": 0,
+                          "uniqueItems": true
+                        }
+                      ]
+                    }
+                  },
+                  "is_acr": {
+                    "$id": "#/properties/validators/items/anyOf/0/properties/is_acr",
+                    "title": "The is_acr schema",
+                    "description": "Indicates whether the notary V1 instance is attached to an ACR (Azure Container Registry) as this requires slight adaptions in communication with the Notary instance.",
+                    "type": "boolean"
+                  },
+                  "cert": {
+                    "$id": "#/properties/validators/items/anyOf/0/properties/cert",
+                    "title": "The cert schema",
+                    "description": "TLS-certificate if using a notary V1 instance requiring a self-signed certificate.",
```

Comment:
```suggestion
                    "description": "TLS certificate if using a notary V1 instance requiring a self-signed certificate.",
```

Reply:
On another note, is this actually only for self-signed or could this also pin the trust root of my cert I got from LE?